partner_daily_notes/models/res_partner.py

```diff
@@ -182,7 +187,7 @@ def _compute_next_contact(self):
             next_contact_type = self.next_contact_type
             res_datetime = datetime.combine(
                 next_contact_date, datetime.strptime(next_contact_time, "%H:%M").time()
-            )
+            ).astimezone(timezone)
```

Comment:
I don't think odoo want's us to store datetimes with other timezones than UTC. I would guess that odoo will ignore this and strip the timezone, but I'm not sure. Have you verified that this does what you expect?